src/main.rs

```diff
@@ -6,14 +6,16 @@ mod account_endpoints;
 mod board;
 mod info_endpoints;
 mod user;
-//#[macro_use] mod macros;
 
 #[tokio::main]
 async fn main() -> Result<(), std::io::Error> {
-    let redis_client = redis::Client::open(env::var("REDIS_URI").unwrap()).unwrap();
+    let redis_client = redis::Client::open(
+        env::var("REDIS_URL").unwrap_or_else(|_| "redis://localhost:6379/1".to_string()),
+    )
+    .expect("Could not find \"REDIS_URL\" environment variable");
```

Comment:
Isnt this already handled?

Reply:
yes, but I changed it to an `expect` to provide more context